utils/ModifyTool/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "ModifyTool": {
+      "commandName": "Project",
+      "commandLineArgs": "d:\\Temp\\MyApp.Client\\MyApp.Client --entity NewModel"
```

Comment:
What happens if there's no D: drive?

Reply:
It's for debug purpose only. Moved launchSettings.json to ignore list.